install_test_deps.sh

```diff
@@ -0,0 +1,6 @@
+# Installs xblock-sdk and dependencies needed to run the tests suite.
+# Run this script inside a fresh virtual environment.
+pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk
+pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt
+pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt
+python setup.py develop
```

Comment:
Why not using a requirements.txt for declaring the xblock-sdk requirement? That take care of the first two lines, no?